addon/index.js

```diff
@@ -0,0 +1,62 @@
+import { isEmpty } from '@ember/utils';
+import { get } from '@ember/object';
+import googletag from 'googletag';
+
+
+export function doTargetingForPath(location=window.location) {
+  doTargeting({
+    'host': location.host,
+    'url': location.pathname,
+    'urlSegments': location.pathname && location.pathname.split('/').filter(segment => segment.length > 0)
+  });
+}
+
+export function clearTargetingForPath() {
```

Comment:
maybe accept a list of keys?

Reply:
If someone only wants to clear specific keys they can use the plain `clearTargeting()` function.